### ISSUE TITLE
ColorRGBSelector no longer showing in 2022.11  

### ISSUE THREAD

**Issue by basnijholt** [home-assistant/frontend#14314]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

In HA 2022.10 I was able to show this color picker in the config flow: https://github.com/basnijholt/adaptive-lighting/pull/329#issuecomment-1233768846:

![image](https://user-images.githubusercontent.com/6897215/200202880-763ac74d-14c4-423e-b907-7fce1f0e2bc4.png)

In my debugging logs I see that the `homeassistant.helpers.selector.ColorRGBSelector` is in my `options_schema`  when creating my `options_flow` with:
```python
        return self.async_show_form(
            step_id="init", data_schema=vol.Schema(options_schema), errors=errors
        )

```

When printing `options_schema` I see this in my logs:
```
2022-11-06 15:34:45.212 INFO (MainThread) [custom_components.adaptive_lighting.config_flow] options_schema: {'lights': <homeassistant.helpers.config_validation.multi_select object at 0x7f379444a4d0>, 'prefer_rgb_color': <class 'bool'>, 'initial_transition': All(Coerce(float, msg=None), Clamp(min=0, max=6553), msg=None), 'sleep_transition': All(Coerce(float, msg=None), Clamp(min=0, max=6553), msg=None), 'transition': All(Coerce(float, msg=None), Clamp(min=0, max=6553), msg=None), 'interval': All(Coerce(int, msg=None), Range(min=0, max=None, min_included=True, max_included=True, msg=None), msg=None), 'min_brightness': All(Coerce(int, msg=None), Range(min=1, max=100, min_included=True, max_included=True, msg=None), msg=None), 'max_brightness': All(Coerce(int, msg=None), Range(min=1, max=100, min_included=True, max_included=True, msg=None), msg=None), 'min_color_temp': All(Coerce(int, msg=None), Range(min=1000, max=10000, min_included=True, max_included=True, msg=None), msg=None), 'max_color_temp': All(Coerce(int, msg=None), Range(min=1000, max=10000, min_included=True, max_included=True, msg=None), msg=None), 'sleep_brightness': All(Coerce(int, msg=None), Range(min=1, max=100, min_included=True, max_included=True, msg=None), msg=None), 'sleep_rgb_or_color_temp': <homeassistant.helpers.selector.SelectSelector object at 0x7f37b83af8b0>, 'sleep_color_temp': All(Coerce(int, msg=None), Range(min=1000, max=10000, min_included=True, max_included=True, msg=None), msg=None), 'sleep_rgb_color': <homeassistant.helpers.selector.ColorRGBSelector object at 0x7f37b83afa90>, 'sunrise_time': <class 'str'>, 'sunrise_offset': <class 'int'>, 'sunset_time': <class 'str'>, 'sunset_offset': <class 'int'>, 'only_once': <class 'bool'>, 'take_over_control': <class 'bool'>, 'detect_non_ha_changes': <class 'bool'>, 'separate_turn_on_commands': <class 'bool'>, 'adapt_delay': All(Coerce(int, msg=None), Range(min=0, max=10000, min_included=True, max_included=True, msg=None), msg=None)}
```

However, the color picker doesn't show anymore:
<img width="541" alt="image" src="https://user-images.githubusercontent.com/6897215/200202915-831a7fa4-5352-413c-bfdf-821032ac7168.png">

Initially reported in https://github.com/basnijholt/adaptive-lighting/issues/353

### Describe the behavior you expected

The color picker to show up.

### Steps to reproduce the issue

1. install https://github.com/basnijholt/adaptive-lighting
2. create a new integration at `.../config/integrations`
3. See that `sleep_rgb_color` is missing. 

### What version of Home Assistant Core has the issue?

2022.11.1

### What was the last working version of Home Assistant Core?

2022.10

### In which browser are you experiencing the issue with?

Chrome, Firefox, Safari

### Which operating system are you using to run this browser?

macOS

**Comment by cjb0001** [comment]
Likewise, all the color pickers that could once be accessed when creating a script with the add service "light: turn_on" are no longer visible in 2022.11, but worked fine 2022.10.

![IMG_8495](https://user-images.githubusercontent.com/102877399/200287827-424201f4-ae47-495a-9c0f-5296b81e3410.jpg)
![homeassistant_scriptcolor_prob](https://user-images.githubusercontent.com/102877399/200287910-2f60c81e-01e9-4ceb-a764-ac88b87bb6c1.png)

**Comment by oNSXo** [comment]
Can confirm that specifically, 2022.10.5 had the RGB picker available and 2022.11.0 caused it to disappear.

**Comment by steverep** [comment]
There was a dynamic import bug that's now fixed.

**Comment by basnijholt** [comment]
Idea any why this fix wasn't included in the latest HA patch release?

It still doesn't work for me at least.

**Comment by steverep** [comment]
Unfortunately core patch releases don't always update the frontend.  The fix looks like it's in 2022.12 beta though.
